firebase/firestore/_utils.py

```diff
@@ -76,13 +76,13 @@ def _decode_datastore(value):
 	elif value.get('booleanValue') is not None:
 		return bool(value['booleanValue'])
 
-	elif value.get('bytesValue'):
+	elif value.get('bytesValue') is not None:
```

Comment:
Could you please provide me an example, where this fails if `is not None` is note used here?

Reply:
Simple. 

If `value.get()` returns integer 0 or float 0.0 then you will `miss` casting to int or float and returning proper value. Unfortunately values equal to zero are treated equal to `False`. But in this case this is a problem because we can store zero value inside Firestore and zero is also valid integer number. So it's better to check always with `None` and treat every not `None` value as valid.